Source/WebCore/Modules/geolocation/GeolocationController.cpp

```diff
@@ -52,7 +52,7 @@ GeolocationController::~GeolocationController()
     // we are supplement of the Page, and our destructor getting called means the page is being
     // torn down.
 
-    auto* client = std::exchange(m_client, nullptr).get();
+    CheckedPtr client = std::exchange(m_client, nullptr);
     client->geolocationDestroyed(); // This will destroy the client.
```

Comment:
The comment here is a bit concerning. If it were true, then we'd crash due to the CheckedPtr that is in scope.

Reply:
Indeed. This was the cause of the debug crashes. Now using `SUPPRESS_...` instead.

Reply:
I commented below why this is happening.